pkg/cloudprovider/aws/amifamily/bootstrap/eksbootstrap.go

```diff
@@ -42,9 +43,12 @@ func (e EKS) Script() string {
 
 	kubeletExtraArgs := strings.Join([]string{e.nodeLabelArg(), e.nodeTaintArg()}, " ")
 
+	if e.isIPv6() {
+		userData.WriteString(" \\\n--ip-family ipv6")
```

Comment:
Is it necessary to pass this if you have an ipv6 clusterdnsip? What is the script doing with this flag?

Reply:
Yes, this flag tells the bootstrap script to register the kubelet w/ the ipv6 node IP discovered via the ipv6 metadata path.